x/stake/keeper.go

```diff
@@ -28,6 +30,17 @@ func NewKeeper(ctx sdk.Context, cdc *wire.Codec, key sdk.StoreKey, ck bank.CoinK
 	return keeper
 }
 
+// InitGenesis - store genesis parameters
+func (k Keeper) InitGenesis(ctx sdk.Context, data json.RawMessage) error {
```

Comment:
Let's add a test function for InitGenesis (which can take a `json.RawMessage` defined in test taken from a hardcoded string)

Reply:
It's a good start to have 
```
	encoded, err := json.Marshal(GenesisState{initialPool(), defaultParams()})
	if err != nil {
		panic(err)
	}
	if err = keeper.InitGenesis(ctx, encoded); err != nil {
		panic(err)
	}
```
But I'd like to see the test with the raw string information (as in the genesis.json file) being turned into the json raw message... this will give us an idea as to what are genesis file will actually look like here

Reply:
Cool, added test with raw string.